src/content/docs/en/install/auto.mdx

```diff
@@ -1,154 +1,420 @@
-  # create a new project with npm
-  npm create astro@latest
-  ```
-  </Fragment>
-  <Fragment slot="pnpm">
-  ```shell
-  # create a new project with pnpm
-  pnpm create astro@latest
-  ```
-  </Fragment>
-  <Fragment slot="yarn">
-  ```shell
-  # create a new project with yarn
-  yarn create astro
-  ```
-  </Fragment>
-</PackageManagerTabs>
-
-You can run `create astro` anywhere on your machine, so there's no need to create a new empty directory for your project before you begin. If you don't have an empty directory yet for your new project, the wizard will help create one for you automatically.
-
-If all goes well, you should see a "Liftoff confirmed. Explore your project!" message followed by some recommended next steps. `cd` into your new project directory to begin using Astro.
-
-If you skipped the `npm install` step during the CLI wizard, then be sure to install your dependencies before continuing.
-
-## 2. Start Astro ✨
-
-Astro comes with a built-in development server that has everything you need for project development. The `astro dev` command will start the local development server so that you can see your new website in action for the very first time.
-
-Every starter template comes with a pre-configured script that will run `astro dev` for you. Use your favorite package manager to run this command and start the Astro development server.
-
-<PackageManagerTabs>
-  <Fragment slot="npm">
-  ```shell
-  npm run dev
-  ```
-  </Fragment>
-  <Fragment slot="pnpm">
-  ```shell
-  pnpm run dev
-  ```
-  </Fragment>
-  <Fragment slot="yarn">
-  ```shell
-  yarn run dev
-  ```
-  </Fragment>
-</PackageManagerTabs>
-
-
-If all goes well, Astro should now be serving your project on [http://localhost:4321/](http://localhost:4321/)!
-
-Astro will listen for live file changes in your `src/` directory, so you will not need to restart the server as you make changes during development.
+### Install from the CLI wizard
 
-When viewing your site in the browser, you'll have access to the [Astro dev toolbar](/en/guides/dev-toolbar/). As you build, it will help you inspect your [islands](/en/concepts/islands/), spot accessibility issues, and more.
+<Steps>
+1. Run the following command in your terminal to start our handy install wizard:
+
+    <Tabs>
+      <TabItem label="npm">
+      ```shell
+      # create a new project with npm
+      npm create astro@latest
+      ```
+      </TabItem>
+      <TabItem label="pnpm">
+      ```shell
+      # create a new project with pnpm
+      pnpm create astro@latest
+      ```
+      </TabItem>
+      <TabItem label="yarn">
+      ```shell
+      # create a new project with yarn
+      yarn create astro
+      ```
+      </TabItem>
+    </Tabs>
+
+    You can run `create astro` anywhere on your machine, so there's no need to create a new empty directory for your project before you begin. If you don't have an empty directory yet for your new project, the wizard will help create one for you automatically.
```

Comment:
```suggestion
    You can run `create astro` anywhere on your machine, so there's no need to create a new empty directory for your project before you begin. If you don't have an empty directory yet for your new project, the wizard will create one for you automatically.
```

Reply:
I'm thinking about this, and it actually does *help* you create one, or at least, it's not correct to say that it creates one automatically. You do need to give it a name in order for it to be created. So, I *think* the original should stand here, because I'm afraid that "automatically" might give a false impression that the user isn't involved.

Reply:
You are right. I was not aware that it allowed the user to enter a full path. I thought it was just the name.